.github/workflows/docker.yml

```diff
@@ -3,24 +3,26 @@ name: Build Base Docker Image
 # Triggered only if environment.yml in master changes
 # Pushes a Docker image with all dev dependencies to Docker Hub
 on:
-  push: 
+  push:
     branches:
       - master
     paths:
       - 'environment.yml'
 
-jobs: 
+jobs:
   build:
-    name: Build base Docker image 
+    name: Build base Docker image
     runs-on: ubuntu-latest
     environment: scipy-dev
     if: "github.repository_owner == 'scipy' && !contains(github.event.head_commit.message, '[ci skip]') && !contains(github.event.head_commit.message, '[skip ci]') && !contains(github.event.head_commit.message, '[skip github]')"
     steps:
       - name: Clone repository
         uses: actions/checkout@v2
-      - name: Lint Docker 
+        with:
```

Comment:
@mckib2 Are these other changes intentional?

Reply:
Some of these are.  Apart from the whitespace-change noise, this makes the git submodule checkout recursive